script.reddit.reader/resources/lib/requests_cache/backends/__init__.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
```

Comment:
probably you can strip requests_cache from your script and make it a standalone script.module. I guess a lot more devs are interested in using it.